<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogZaci</class>
 <widget class="QDialog" name="DialogZaci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zaci</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DialogZaci {
background: rgb(204,229,250);
}
#pushButtonHledat{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonVsechny{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonPridatZak{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonSmazatZak{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}
#pushButtonUpravit{
background: rgb(235,235,235);
border-radius : 20px;
border : 3px solid rgb( 0,76,153);
}







</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButtonUpravit">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButtonUpravit{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>         Výpůjčka                  </string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButtonSmazatZak">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButtonSmazatZak{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>        Smazat žáka               </string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonVsechny">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButtonVsechny{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>  Výpis všech žáků</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonHledat">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButtonHledat{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>Hledat žáka podle příjmení</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButtonPridatZak">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#pushButtonPridatZak{
color: rgb(0,67,134)
}</string>
     </property>
     <property name="text">
      <string>      Přidat žáka   </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
